hydra-node/src/Hydra/API/HTTPServer.hs

```diff
@@ -187,40 +133,71 @@ httpApp tracer directChain pparams getInitializingHeadId request respond = do
 --
 -- @
 --
--- // Committing public key output
+-- // FullCommitRequest
```

Comment:
why do we keep this long comment here?
is not enough having the api.yaml?

Reply:
We had it also before for the haskell people looking at haddocks.